runtimeverification/mir-semantics
# Refactor test data in preparation for implementing semantics

This PR introduces changes in how test data is handled:

* fail files are now .tsv files and include the exit code
* add Makefile targets to generate the fail files
* move parsing tests to test-data/handrwitten-mir/parsing
* make sure parsing tests are not tested for execution, because they don't always make sense
* add infrastructure for handwritten execution tests
* minor refactoring in `mir-syntax.k`

I found these changes to be helpful while exploring how to implement the execution semantics. The changes alter the format for the fail files and thus produce massive diffs, hence it would be nice to have them merged before the semantics PR to minimize the noise.
